changelog/selections.dd

```diff
@@ -0,0 +1,28 @@
+`dub select`, `dub deselect`, `dub upgrade -l`
+
+DUB now exposes CLI commands to interact with the dub.selections.json file:
+
+`dub select` can be used to select or change a dependency inside
+dub.selections.json.
+
+`dub deselect` can be used to remove a dependency from dub.selections.json.
+
+`dub upgrade --list` (or `-l`) now does a dry-run and lists all the existing
+packages + packages to upgrade in an easy-to-read interface. This can be
+combined with `--missing-only` (now available as `-m` as well) to show what is
+being done internally when running `dub build`. Both an upgrade as well as
+listing can be done by running `dub upgrade --list --dry-run=false` or
```

Comment:
I'm a bit bothered by the fact we now have more switches to do the same thing, slightly differently.
We have `dub -v` and `dub upgrade -l` and `-l` implies dry run but we can also use `-d false`...
How about simplifying it and always having `dub upgrade -l` behavior, and just keeping dry-run for `dub upgrade -l` ?
Note that `dub upgrade -l` is not IMO very consistent CLI behavior: options (things starting with dash) shouldn't be used to implement subcommands, they should only slightly modify command behavior. So if you want to keep the `dub upgrade -l` behavior I would have gone with `dub upgrade list`. I'm not attached to the specifics, so happy to hash this out.